### PR TITLE
Fix bot message that makes users think they did something wrong

### PR DESCRIPTION
**Pull request by davetcoleman**
> Thanks for opening this pull request! Please check out our contributing guidelines.

This was the default message from the bot documentation, but made people here at WMD think they did something wrong.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: >
-  Thanks for opening your first issue here! Be sure to follow the issue template!
+  Thanks for opening your first issue here!
```
Comment (rhaschke) [inline .github/config.yml]:
```suggestion
  Thanks for reporting an issue. We will have a look asap. 
  If you can think of a fix, please consider providing it as a pull request.
```

File: .github/config.yml [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: >
-  Thanks for opening this pull request! Please check out our contributing guidelines.
+  Thanks for opening your first MoveIt! pull request :robot:
```
Comment (rhaschke) [inline .github/config.yml]:
```suggestion
  Thanks for helping in improving MoveIt!
```

**Comment by davetcoleman** [comment]
> I think this message is also issued for frequent github users, isn't it?

No I'm pretty sure only if you're new to the repo

File: .github/config.yml (hunk 1 above)
Comment (davetcoleman) [reply .github/config.yml]: I like this suggestion!

**Pull request merged**